src/style/sass/main.scss

```diff
@@ -20,86 +20,130 @@ body {
+                    }
+            
+                    > .tweetContainer {
             
-                        p {
-                            margin-bottom: 3px;
+                        &:nth-of-type(odd) {
+                            background-color: #e8f5fd;
                         }
             
-                        .name {
-                            @include styled-font($weight: bold);
-                            //font-weight: bold;
+                        &:nth-of-type(even) {
+                            background-color: #e9e9e9;
                         }
             
-                        .handle {
-                            @include styled-font($size: 8pt);
-                            // font-size: 8pt;
+                        &:not(:last-of-type) {
+                            border-bottom: 1px solid #d1d2d2;
                         }
                     }
-        
-                    span {
-                        display: block;
-                        //font-size: 8pt;
-                        margin-bottom: 5px;
-                        @include styled-font($style: italic, $size: 8pt)
-                        //font-style: italic;
-                    }
-        
-                    a, a:visited {
-                        color: black;
-                        text-decoration: none;
+            
+                    .tweetContainer {
+                        padding: 10px 20px;
+                        display: flex;
+                        flex-direction: row;
+                        align-items: center;
+            
+                        .userDiv {
+                            text-align: center;
+                            min-width: 175px;
+                            max-width: 175px;
+                            height: 100%;
+                            margin-right: 20px;
+                
+                            img {
+                                max-width: 85px;
+                                height: auto;
+                                margin: 10px 0 5px;
+                                border-radius: 40px;
+                            }
+                
+                            p {
+                                margin-bottom: 3px;
+                            }
+                
+                            .name {
+                                @include styled-font($weight: bold);
+                            }
+                
+                            .handle {
+                                @include styled-font($size: 8pt);
+                            }
+                        }
+
+                        .textDiv {
+                            span {
+                                display: block;
+                                margin-bottom: 5px;
+                                @include styled-font($style: italic, $size: 8pt)
+                            }
+                
+                            a, a:visited {
```

Comment:
don't use generic tags for css unless you absolutely need it.  Generally not a good practice.